docs/source/control_flow_cond.rst

```diff
@@ -0,0 +1,150 @@
+Control Flow - Cond
+====================
+
+Export supports the structured control flow operator `torch.ops.higher_order.cond`. It can be used to specify if-else like control flow within
+some code.
+
+torch.ops.higher_order.cond
```

Comment:
Do we really need to document this? Ideally we would replace this with torch.cond, but that didn't make the branch cut. In the future we want people to use torch.cond instead of directly using torch.ops.higher_order.cond.

If we must document this, then there needs to be a big note somewhere that this API is "prototype" as a part of the prototype torch.export release.

Reply:
Yeah, these are valid points. I also want to add that people will be confused about which api to use. Ideally, there should be one and only one way to do conditional, that is, torch.cond. When torch.cond is ready, we can have a documentation for the API torch.cond and an IR node specification for torch.ops.higher_order.cond so that users can do further transformations/lowering.  For now, it might do more harm if we release a documentation that directly uses torch.ops.higher_order.cond . 

What do you think? @gmagogsfm

Reply:
> Do we really need to document this?

It depends on whether there are flaws with the part of cond implementation that made branch cut. Will there be adverse effects (other than API confusion) if we make `torch.ops.higher_order.cond` public? 

> If we must document this, then there needs to be a big note somewhere that this API is "prototype" as a part of the prototype torch.export release.
Agreed